commit 6a3fed65b85a4bc592265dde0d74d6d7a3185543 (refs/changes/90/2890/2)
Author: Richard Kiene <richard.kiene@joyent.com>
Date:   2017-10-31T19:48:54+00:00 (1 year, 11 months ago)
    
    CNS-216 Only fetch CNs that are setup
    Reviewed by: Todd Whiteman <todd.whiteman@joyent.com>
    Approved by: Todd Whiteman <todd.whiteman@joyent.com>

diff --git a/lib/cn-filter.js b/lib/cn-filter.js
index c0231e7..7716bd5 100644
--- a/lib/cn-filter.js
+++ b/lib/cn-filter.js
@@ -61,7 +61,7 @@ util.inherits(CNFilter, stream.Transform);
 
 CNFilter.prototype.updateServers = function (cb) {
 	var self = this;
-	self.client.get('/servers?extras=status',
+	self.client.get('/servers?setup=true&extras=status',
 	    function (err, req, res, objs) {
 		if (err) {
 			self.log.warn(err,
